### DISCUSSION TITLE
Maintainer Appreciation Thread

### DISCUSSION THREAD

**Discussion by subdavis** [Leseratte10/acsm-calibre-plugin#15] [General]
Excellent work!

I am a desktop Linux user who cannot use Adobe Digital Editions (WINE is :vomiting_face:)

So for the last two years, I've been the maintainer of https://github.com/subdavis/kobo-book-downloader, which quite a few people seem to have found useful.   I recently found your rplugin though, and it works flawlessly.  I'm so thrilled that I've updated the README to recommend people just use this instead, and I hope I can just stop maintaining my thing.

I'll try and keep up with goings-on here, and I'm looking forward to poking around to see how this actually works.

Thank you!

**Comment by p1nkyy** [comment]
Thanks so much for your awesome work. I found this accidentally and immediately tried it. ADE has been such a PITA for me, it keeps freezing, it won't retain the books I've processed, maxed at 100 books and it won't open again, always need to force close via task manager, etc etc. This is a godsend literally!

**Comment by HMLordie** [comment]
This is amazing work! I was on Windows 11 and after some upgrades, my ADE 4.5 totally stopped working and would not open. Sourced around and found your thread from mobilereads forum and led me to this plugin.

Non geeks may find it a bit difficult to install the plugin but if you have used Calibre and imported other DRM plugins, the instructions given are just fine.

Amazing works and thanks! In fact it works even better than ADE due to the slowness in returning loan books. Just a suggestion though, to add a shortcut to open the plugin to return loan books or a mass return to Library would be an even better upgrade!

**Reply by Leseratte10** [reply]
Thanks for your comment. The most recent version of the plugin has an icon that you can add to Calibre's menu bar, when you click on that you can choose "Show loaned books" to immediately open the list of loaned books. 

The easiest way to install it is probably through Calibre's plugin index (other than DeDRM, this plugin is available on there). Unfortunately I haven't had much time for development recently so the plugin is still stuck in "version limbo" with 0.0.16 being available on Mobileread and through the Plugin updater and 0.1.0 available on Github only..

**Comment by nydragon** [comment]
Even if Adobe Digital Edition would work through wine, it is substantially nicer to go through Calibre with this plugin installed

**Comment by doctor-beat** [comment]
Brilliant plugin.

**Comment by SrNetoChan** [comment]
This just saved me from using a Windows VM everytime, using an old ade executable (4.x doesn't work for me) everytime I wanted to autorize and upload a book to my ereader (a kobo device). Many thanks @Leseratte10 for your work!!
